tools/build_config/clone_herbert_branch.ps1

```diff
@@ -0,0 +1,26 @@
+# Clone and build the given Herbert branch. If no branch is given, default to
+# master
+# Pass a specific branch using the -branch parameter, any other arguments are
+# passed verbatim to the Herbert build script
+
+param(
+  [string][Alias("B")]$branch = "master",
+  [string]$build_args = ""
+)
+
+. $PSScriptRoot/powershell_helpers.ps1 <# Imports:
+  Write-And-Invoke
+  Invoke-In-Dir
+#>
+
+$HERBERT_URL = "https://github.com/pace-neutrons/Herbert.git"
+
+Write-Output "Cloning and building Herbert branch $branch..."
```

Comment:
should this be a different Write in the two branches: it's not cloning if there's a Herbert folder already there

Reply:
This print statement is more about telling you want branch is being built. I've removed the cloning bit. What action is actually being performed is made obvious by the commands being printed before they're executed